src/logics/stargate/configs.ts

```diff
@@ -0,0 +1,537 @@
+import { arbitrumTokens, avalancheTokens, mainnetTokens, metisTokens, optimismTokens, polygonTokens } from './tokens';
+import * as common from '@protocolink/common';
+
+type ContractNames = 'Router' | 'RouterETH' | 'FeeLibrary' | 'LayerZeroEndpoint';
+
+export enum ChainId {
+  ETHEREUM = 101,
+  AVALANCHE = 106,
+  POLYGON = 109,
+  ARBITRUM = 110,
+  OPTIMISM = 111,
+  METIS = 151,
+  BASE = 184,
+}
+
+export enum PoolId {
+  USDC = 1,
+  USDT = 2,
+  DAI = 3,
+  FRAX = 7,
+  USDD = 11,
+  ETH = 13,
+  sUSD = 14,
+  LUSD = 15,
+  MAI = 16,
+  METIS = 17,
+  metisUSDT = 19,
+  mETH = 22,
+}
+
+export interface Pool {
+  id: number;
+  token: common.Token;
+  decimals: number;
+  paths: { chainId: number; poolIds: number[] }[];
+}
+
+export interface PoolConfig extends Pool {
+  chainId: number;
+}
+
+export interface Config {
+  id: number;
+  chainId: number;
+  contract: Record<ContractNames, string>;
+  pools: Pool[];
+  STG?: common.Token;
+}
+
+export const configs: Config[] = [
+  {
+    id: ChainId.ETHEREUM,
+    chainId: common.ChainId.mainnet,
+    contract: {
+      Router: '0x8731d54E9D02c286767d56ac03e8037C07e01e98',
+      RouterETH: '0x150f94B44927F078737562f0fcF3C95c01Cc2376',
+      FeeLibrary: '0x6FaF1AB85FFbe7B3A557F4864046ff861734afd0',
+      LayerZeroEndpoint: '0x66A71Dcef29A0fFBDBE3c6a460a3B5BC225Cd675',
+    },
+    STG: mainnetTokens.STG,
+    pools: [
+      {
+        id: PoolId.USDC,
+        token: mainnetTokens.USDC,
+        decimals: 6,
+        paths: [
+          { chainId: common.ChainId.optimism, poolIds: [PoolId.USDC] },
+          { chainId: common.ChainId.polygon, poolIds: [PoolId.USDC, PoolId.USDT] },
+          { chainId: common.ChainId.arbitrum, poolIds: [PoolId.USDC, PoolId.USDT] },
+          { chainId: common.ChainId.avalanche, poolIds: [PoolId.USDC, PoolId.USDT] },
+        ],
+      },
+      {
+        id: PoolId.USDT,
+        token: mainnetTokens.USDT,
+        decimals: 6,
+        paths: [
+          { chainId: common.ChainId.optimism, poolIds: [PoolId.USDC] },
+          { chainId: common.ChainId.polygon, poolIds: [PoolId.USDC, PoolId.USDT] },
```

Comment:
Curious that if it's same between `[PoolId.USDC, PoolId.USDT]` and `[PoolId.USDT, PoolId.USDC]`, as furucombo-interface is `poolIds: [2, 1]`

Reply:
IMO, they are the same. But we can double-check this one with the author.

Reply:
The order will only affect the display of the Furucombo interface. In our interface, we assume that the user only wants to bridge, not bridge and swap. So we can find out that the first pool in the poolIds always matches the token.

Reply:
The order will only affect the display of the Furucombo interface. In our interface, we assume that the user only wants to bridge, not bridge and swap. So we can find out that the first pool in the poolIds always matches the token.

Reply:
@Jeff-CCH I think we can follow the same approach as Furucombo as the UI may use the same logic to display the token list after migration.